packages/cli/src/oclif/commands/register.js

```diff
@@ -132,6 +132,13 @@ class RegisterCommand extends ZapierBaseCommand {
       this.startSpinner('Retrieving details for your integration');
       this.app = await getWritableApp();
       this.stopSpinner();
+
+      // Block non-private apps from updating settings
+      if (this.app?.intention !== 'private') {
```

Comment:
Should we use the `status` field instead?

```suggestion
      if (this.app?.status !== 'private') {
```

Reply:
@eliangcs Great catch - I was unaware of how `app.status` is being used. I checked [the logic we're using in VB](https://gitlab.com/zapier/team-developer-platform/dev-platform/-/blob/9fa28d8bacd04ebdad5937bd039c71aede4ede47/web/frontend/assets/app/entities/CliApp/CliApp.ts#L96) to limit users from updating app settings for published apps and decided to implement that logic in 576b463ae49a1007ce4014c28d62bb36e26adbfd.

Can you please re-review?

Reply:
Thanks for checking the logic on the UI side! Looks good to me!

Reply:
@eliangcs Thanks for reviewing! I saw that you pushed ba6ca25072f70d22c138e8e7d60e8ab9a26f2bbf, but decided to revert it - is there a more ideal solution than what I've pushed?

Reply:
@gregilo I thought the `public_ish` field returned by the server is defined as `status in ('public', 'beta')` and we could use it, but then I found out `public_ish` is in fact defined as `status == 'public'`, so I reverted it.